src/js.rs

```diff
@@ -5,7 +5,7 @@
 // <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
-use crate::{util::uninit_slice_fill_zero, Error};
```

Comment:
After this change, the only other place which uses `uninit_slice_fill_zero` is the custom backend. It could be worth to inline this function there.

Reply:
Ya I was going to clean this up when I cleaned up the `custom` implementation.